### PR TITLE
Consistency & duplicate pointer

### PR DESCRIPTION
**Pull request by ghost**
Yep.

### CODE REVIEW

File: BigBaseV2/src/pointers.cpp [hunk 1]
```diff
@@ -365,7 +362,8 @@ namespace big
 
 		/**
 		 * Freemode thread restorer through VM patch
-		 */
+		*/
+
```
Comment (Yimura) [inline BigBaseV2/src/pointers.cpp:366]: Why this random newline?

File: BigBaseV2/src/pointers.cpp [hunk 2]
```diff
@@ -393,6 +391,7 @@ namespace big
 		}
 
 		m_hwnd = FindWindowW(L"grcWindow", nullptr);
+
```
Comment (Yimura) [inline BigBaseV2/src/pointers.cpp:394]: Why this random newline?

**Review by Yimura — CHANGES_REQUESTED** [review]

File: BigBaseV2/src/pointers.cpp (hunk 1 above)
Comment (ghost) [reply BigBaseV2/src/pointers.cpp:366]: Proper formatting = legibility

File: BigBaseV2/src/pointers.cpp (hunk 2 above)
Comment (ghost) [reply BigBaseV2/src/pointers.cpp:394]: Proper formatting = legibility

**Review by Yimura — APPROVED** [review]

**Pull request merged**